docs/tctl/workflow/reset.md

```diff
@@ -8,11 +8,13 @@ tags:
   - tctl
 ---
 
-The `tctl workflow reset` command resets a [Workflow Execution](/concepts/what-is-a-workflow-execution) by either `eventId` or `resetType`.
+The `tctl workflow reset` command resets a [Workflow Execution](/concepts/what-is-a-workflow-execution) by either [`eventId`](#eventid)or [`resetType`](#resettype).
 
-See also [`tctl workflow reset-batch`](/tctl/workflow/reset-batch).
+Resetting a Workflow allows the process to be resumed from a certain point without losing your parameters or Work History.
 
-`tctl workflow reset [<modifiers>]`
+To run multiple Reset operations at once, see [`tctl workflow reset-batch`](/tctl/workflow/reset-batch).
```

Comment:
I was going to say "Reset is not a core term, let's not capitalize..." - but I am actually thinking maybe it is...
Might want to make a note in the Docs project plan to explain this term -